smurfs/src/actions/index.js

```diff
@@ -13,3 +13,72 @@
    U - updateSmurf
    D - deleteSmurf
 */
+
+import axios from "axios";
+
+export const FETCH_SMURFS = "FETCH_SMURFS";
+export const FETCH_SMURFS_SUCCESS = "FETCH_SMURFS_SUCCESS";
+export const FETCH_SMURFS_FAILURE = "FETCH_SMURFS_FAILURE";
+export const CREATE_SMURF = "CREATE_SMURF";
+export const ADD_SMURF_SUCCESS = "ADD_SMURF_SUCCESS";
+export const ADD_SMURF_FAILURE = "ADD_SMURF_FAILURE";
+export const DELETE_SMURF_SUCCESS = "DELETE_SMURF_SUCCESS";
+export const DELETE_SMURF_FAILURE = "DELETE_SMURF_FAILURE";
+
+export const getSmurfs = () => dispatch => {
+  dispatch({
+    type: FETCH_SMURFS
+  });
+  axios
+    .get("http://localhost:3333/smurfs")
+    .then(response => {
+      dispatch({
+        type: FETCH_SMURFS_SUCCESS,
+        payload: response.data
+      });
+    })
+    .catch(error => {
+      dispatch({
+        type: FETCH_SMURFS_FAILURE,
+        payload: error
+      });
```

Comment:
Might want to do error.response to see if that gives you a better indication of what the error is as opposed to just `Error {}`